app/views/open_source/index.erb

```diff
@@ -1,21 +1,32 @@
-<div class="col-md-9 p-5">
-  <span><b>32</b> open source projects:</span>
-  <li><b>16</b> ruby projects</li>
-  <li><b>8</b> ruby projects</li>
-  <li><b>14</b> react projects</li>
-  <li><b>4</b> iOS projects</li>
+<div class="metrics-container">
+  <div class="p-5">
+    <span><b>32</b> open source projects:</span>
+    <li><b>16</b> ruby projects</li>
+    <li><b>8</b> ruby projects</li>
+    <li><b>14</b> react projects</li>
+    <li><b>4</b> iOS projects</li>
+  </div>
+  <div class="main-container">
+    <div class='col-md-9 p-3'>
+      <%= line_chart @visits_per_language.all_datasets, ytitle: "Repository Unique Visits" %>
+    </div>
+  </div>
+  <div class="main-container">
+    <div class='p-3 fl-3'>
+      <%= line_chart [{}], ytitle: "Repository Visits" %>
+    </div>
+    <div class='metrics-summary-group fl-1'>
+      <span class='metrics-summary-item'>800 visits this month</span>
+      <span class='metrics-summary-item'>3500 visits this year</span>
+    </div>
+  </div>
+  <div class="main-container">
+    <div class='p-3 fl-3'>
+      <%= line_chart [{}], ytitle: "Stars" %>
+    </div>
+    <div class='metrics-summary-group fl-1'>
+      <span class='metrics-summary-item'>56 new stars this month</span>
+      <span class='metrics-summary-item'>3600 total stars</span>
+    </div>
+  </div>
```

Comment:
It's better to put some meaningful name class than `fl-1` or `p-3`, is easy to read and find the style. Also, try to see as a component, if you need specific styles for a view, creates a CSS for the view just to change what you need, For example:
```scss
# file of generic style
.main-container {
  ...
}

.metrics-summary-group {
  ...
}

# File of the view
.main-container {
  .metrics-graph {
    flex: 3;
    padding: 5px;
  }
  .metrics-summary-group {
    flex: 1;
  }
}
```
Any time that you want another attribute you will create a class with that attribute so what is better:
1. `fl-1 p-3 m-5 c-5`
2. `filter-section`

Try to get this one with that approach and then you will see the other views better and with clean code. Important to be synchronized with the team about that 😃